src/editorial/web/components/quote-icon/stories.tsx

```diff
@@ -0,0 +1,422 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+import { Design, Display, Pillar, Special } from '@guardian/types';
+import { headline } from '@guardian/src-foundations/typography';
+
+import { QuoteIcon } from './index';
+
+export default {
+	title: 'Editorial/QuoteIcon',
+	component: QuoteIcon,
+};
+
+const SmallHeadline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			${headline.xxxsmall()};
+			display: inline;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const Headline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			${headline.xxxsmall()};
+			display: inline;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const LargeHeadline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			${headline.xsmall()};
+			display: inline;
+		`}
+	>
+		{children}
+	</div>
+);
+
+export const Primary = () => (
+	<>
+		<h1>Tiny</h1>
+		<div>
+			<QuoteIcon
+				size="tiny"
+				format={{
+					display: Display.Standard,
+					design: Design.Article,
+					theme: Pillar.News,
+				}}
+			/>
+			<span>
+				I look life a buffoon. I feel incredible. And then I vomit
```

Comment:
I think this is sound from an SVG perspective.

```suggestion
				I look like a buffoon. I feel incredible. And then I vomit
```